### PR TITLE
assets: add UI contrib components

### PR DESCRIPTION
**Pull request by kpsherva**
* closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1276

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -82,3 +89,15 @@ jobs:
       - name: Run tests
         run: |
           ./run-tests.sh
+
+      - name: Install deps for frontend tests
+        working-directory: ./invenio_administration/assets/semantic-ui/js/invenio_administration
```
Comment (alejandromumo) [inline .github/workflows/tests.yml:94]: is this directory correct (`invenio_administration`)?  Same question applies to the tasks below

File: invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/.eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
```
Comment (alejandromumo) [inline invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/.eslintrc.yml:4]:
```suggestion
# Copyright (C) 2022-2023 CERN.
```

There are multiple copyright headers that need to be changed.

File: invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// This file is part of InvenioAdministration
```
Comment (alejandromumo) [inline invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/index.js:1]:
```suggestion
// This file is part of Invenio
```

File: invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/AwardResults.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
+// This file is part of React-Invenio-Deposit
```
Comment (alejandromumo) [inline invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/AwardResults.js:1]:
```suggestion
// This file is part of Invenio
```

File: invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/AwardResults.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+// Copyright (C) 2021 CERN.
+// Copyright (C) 2021 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
```
Comment (alejandromumo) [inline invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/AwardResults.js:5]:
```suggestion
// Invenio-Vocabularies is free software; you can redistribute it and/or modify it
```

**Review by alejandromumo — APPROVED** [review]
LGTM.

minor: since we're moving code around, there are multiple references to `React-Invenio-Deposit` or `Invenio-Administration` in the copyrights headers.  Also outdated headers date (2021, 2022, etc)

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -55,6 +55,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Use Node.js ${{ matrix.node-version }}
```
Comment (ntarocco) [inline .github/workflows/tests.yml:58]: missing `node-version` in the matrix above?

**Review by ntarocco — APPROVED** [review]

**Pull request merged**